### PR TITLE
Add a linter

### PR DESCRIPTION
**Pull request by hitohata**
## Overviews of implementation

Added the Biome.

## Review points

* The biome's setting is the default setting. 
* I ran it to adapt suggestions, so the diff got huge.

### CODE REVIEW

**Comment by taniguchiiqqq** [comment]
I verified that the Readme and the formatter work. In my opinion, a codebase should share one format rule so that files in the cdk dir should also be formatted. How about adding the linter into the cdk dir?

**Comment by hitohata** [comment]
@taniguchiiqqq 
Thank you for your comment.
The biome can share a single config file. So, I'll move the config file from the backend directory to tha project top.

https://biomejs.dev/reference/configuration/

Do you think shoud we run formatting on the project top so that addapt both of them? Or, run it in the each directry?

**Comment by taniguchiiqqq** [comment]
@hitohata 
> Do you think shoud we run formatting on the project top so that addapt both of them?
I would love to adopt this idea.

**Comment by hitohata** [comment]
@taniguchiiqqq 

> Do you think shoud we run formatting on the project top so that addapt both of them?

I thought I could accomplish it using the *include* option. 

https://biomejs.dev/reference/configuration/#filesinclude

However, I ran into an error that says there is no option of *include*.  And I realized it is a new feature and available at a nightly release.

https://github.com/biomejs/biome/releases/tag/cli%2Fv1.2.2-nightly.1e3fc94

Hence, I think we have to set the target file in the command for a while.
I'll move the config file and write the command with the selected target directory on the README or something.

**Comment by taniguchiiqqq** [comment]
Thank you for your explanation. I understand what the problem is.
I found we can install a nightly release with npm now.
https://www.npmjs.com/package/@biomejs/biome/v/latest?activeTab=versions
I couldn't get how the "include" option works, but the following code worked on my local. Please check it and decide if it is acceptable to use the nightly release version.
```json
  "scripts": {
~~
    "format": "biome format --write ../../"
  },
  "devDependencies": {
    "@biomejs/biome": "1.2.2-nightly.ee2e2ab",
~~
}
```

```json
{
~~
	"linter": {
~~
	},
	"files": {
		"ignore": ["**/*.js","**/*.json"]
	}
```

**Comment by hitohata** [comment]
@taniguchiiqqq 
I've updated the diome.json and README.md of the top page.
When you run that command written in the README.md, you can lint the code.

File: backend/main/biome.json
Comment (taniguchiiqqq) [inline backend/main/biome.json:1]: This file seems no longer needed.

**Review by taniguchiiqqq — APPROVED** [review]
LGTM

**Pull request merged**